src/main/webapp/WEB-INF/jsp/editDisplayPreferences.jsp

```diff
@@ -112,12 +112,12 @@
                 <div class="col-md-12">
                     <c:choose>
                         <c:when test="${not prefHideAbstract}">
-                            <input type="checkbox" value="true" name="hideAbstract"/>
-                            <label><spring:message code="edit.pref.hideabstract"/></label>
+                            <input role="checkbox" aria-labelledby="HideAbstract" type="checkbox" value="true" name="hideAbstract"/>
+                            <label id="HideAbstract"><spring:message code="edit.pref.hideabstract"/></label>
```

Comment:
:eye_speech_bubble:  should the `<label>` wrap both the `<input>` and the `<spring:message>`?